interbotix_ros_xscobots/interbotix_xscobot_moveit/CMakeLists.txt

```diff
@@ -8,4 +8,7 @@ ament_package()
 install(DIRECTORY launch DESTINATION share/${PROJECT_NAME}
   PATTERN "setup_assistant.launch" EXCLUDE)
 install(DIRECTORY config DESTINATION share/${PROJECT_NAME})
+install(
+  PROGRAMS scripts/interbotix_moveit_example.py
+  DESTINATION lib/${PROJECT_NAME})
```

Comment:
```suggestion
install(
  PROGRAMS
    scripts/interbotix_moveit_example.py
  DESTINATION
    lib/${PROJECT_NAME}
)
```

Reply:
The changes you made to this file do not reflect the suggestions.

Reply:
Does it reflect now? There has been a lot of confusion on tracking this suggestion in vscode. Can you create a new comment if its still not fixed?